.github/workflows/prod.yml

```diff
@@ -13,7 +13,6 @@ jobs:
       matrix:
         target:
           [
-            { os: ubuntu, version: 18.04, dist: bionic, package-type: DEB },
```

Comment:
Does this mean we are officially dropping support for Ubuntu 18.04 packages?

Reply:
If so, we need to update the required checks in our branch protection rules for `main` to remove the 2 Ubuntu 18.04 checks that aren't running anymore and add the new Mariner/Ubuntu 22.04 ones.